sdk/iot/pnp/src/az_iot_pnp_client.c

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#include <stdint.h>
+
+#include "az_iot_pnp_client.h"
+#include <az_precondition_internal.h>
+#include <az_result.h>
+#include <az_span.h>
+
+#include <_az_cfg.h>
+
+static const uint8_t pnp_client_param_separator = '&';
+static const uint8_t pnp_client_param_equals = '=';
+
+static const az_span pnp_model_id = AZ_SPAN_LITERAL_FROM_STR("digital-twin-model-id");
+
+AZ_NODISCARD az_iot_pnp_client_options az_iot_pnp_client_options_default()
+{
+  az_iot_pnp_client_options options;
```

Comment:
We usually use literal init syntax for this. Example: https://github.com/Azure/azure-sdk-for-c/blob/master/sdk/storage/blobs/src/az_storage_blobs_blob_client.c#L34

Reply:
Changed.